server/service/src/sync/program_schemas/hiv_care_encounter_ui_schema.json

```diff
@@ -5,6 +5,33 @@
     "data": 1
   },
   "elements": [
+    {
+      "type": "Category",
+      "label": "Visit Type",
+      "options": {
+        "icon": "visit"
+      },
+      "elements": [
+        {
+          "type": "Control",
+          "label": "Visit Type:",
+          "scope": "#/properties/visit/properties/visitType"
+        },
+        {
+          "type": "Control",
+          "scope": "#/properties/visit/properties/location",
+          "rule": {
+            "effect": "ENABLE",
+            "condition": {
+              "scope": "#/properties/visit/properties/visitType",
+              "schema": {
+                "enum": ["Outreach", "Refill"]
+              }
```

Comment:
I wonder if this should be "VISIBLE" rather than "ENABLE"? Cos if you enter something in "Location", then change the type to "Standard", the location stays but its no longer editable. Would it look better to just hide it?

![Screenshot 2023-03-15 at 9 25 47 AM](https://user-images.githubusercontent.com/5456533/225128390-e5d4a257-856d-4cc7-9ae1-44a8f9baa89c.png)